assets/src/js/components/Explorer.vue

```diff
@@ -140,7 +136,22 @@ export default {
     updateSearchResults: function (event) {
       this.searchResults = event;
     }
+  },
+
+  mounted: function () {
```

Comment:
This makes me a sad panda.

Reply:
My deepest apologies sad 🐼, I've extracted the map into its own component. I hope this improves things a bit.